Sender_stream.cpp

```diff
@@ -0,0 +1,25 @@
+// Test driven Development. -> Define the testcase before implementation.
+#define CATCH_CONFIG_MAIN 
+
+#include "testsender/catch.hpp"
+#include "Sender/Sender_Stream.h"
+
+
+TEST_CASE("Testcase to check input data from file") 
```

Comment:
No tests for the output format? This is the end responsibility of the Sender.
The absence of tests here means that future mistakes in the output will only be caught when integrating with the receiver. That will take a lot more time than a simple unit-test